src/util/util.h

```diff
@@ -8,7 +8,9 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
+#include <strings.h>
```

Comment:
Hmm, `strings.h` doesn't exist in MSVC build.

Reply:
Yeah, no idea why it was there actually.